### PR TITLE
Create zipcode_rate_archives table

### PR DESCRIPTION
**Pull request by devin-mccurdy**
Database migration PR for https://github.com/department-of-veterans-affairs/gibct-data-service/pull/439

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Review by zurbergram — APPROVED** [review]
LGTM

File: db/migrate/20190916105800_create_zipcode_rates_archive.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class CreateZipcodeRatesArchive < ActiveRecord::Migration
+  def up
+    safety_assured do
+      execute "create table zipcode_rate_archives (like zipcode_rates
```
Comment (jholton) [inline db/migrate/20190916105800_create_zipcode_rates_archive.rb]: For the sake of consistency, can we call this table `zipcode_rates_archives`?
We archive institutions to `institutions_archives`, not `institution_archives`.

**Review by jholton — APPROVED** [review]
LGTM

File: db/migrate/20190916105800_create_zipcode_rates_archives.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class CreateZipcodeRatesArchives < ActiveRecord::Migration
+  def up
+    safety_assured do
```
Comment (omgitsbillryan) [inline db/migrate/20190916105800_create_zipcode_rates_archives.rb]: I'm a little late - j/w what `strong_migrations` was complaining about that made you wrap with `saftey_assured`?

Comment (dmccurdybah) [reply db/migrate/20190916105800_create_zipcode_rates_archives.rb]: `strong_migrations` doesn't inspect direct SQL statements so will flag it as a "Possibly dangerous operation" and tell you to think carefully about your life choices before wrapping it w/ `safety_assured`

**Pull request merged**